.travis.yml

```diff
@@ -5,16 +5,22 @@ before_script:
     - bash /tmp/bats/install.sh /tmp/local
     - export PATH=$PATH:/tmp/local/bin
     # install xspec
-    - git clone -b master https://github.com/xspec/xspec.git /tmp/xspec
-    # install saxon
+    - git clone -b $TRAVIS_BRANCH https://github.com/xspec/xspec.git /tmp/xspec
```

Comment:
When this change is merged into a branch in a forked repository (a `foo` branch of my forked `xspec` repo, for instance), Travis fails:

```
$ git clone -b $TRAVIS_BRANCH https://github.com/xspec/xspec.git /tmp/xspec
Cloning into '/tmp/xspec'...
fatal: Remote branch foo not found in upstream origin
```

because `foo` branch being tested exists only in my forked `xspec` repository, not in `https://github.com/xspec/xspec.git`.

I wonder why this `git clone` line is necessary in the first place.
When this line is executed on Travis, the repo branch being tested is already cloned to the current directory.
This `git clone` line is just a residue of a certain past thing, perhaps?